controllers/entry.js

```diff
@@ -1,26 +1,28 @@
+const { userIsAdmin } = require("./util/auth");
```

Comment:
this looks like a duplicate import with the one on line 9. is there a reason for this? if not this can be removed